### PR TITLE
GET review-comment_counter

### PR DESCRIPTION
**Pull request by adnaani**

### CODE REVIEW

File: model/reviews.js [hunk 1]
```diff
@@ -3,11 +3,17 @@ const db = require("../db/connection");
...
+      FROM reviews
+      LEFT JOIN comments
+      ON comments.review_id = reviews.review_id
+      WHERE reviews.review_id = $1
+      GROUP BY reviews.review_id
+      `,
       [review_id]
     )
     .then(({ rows }) => {
+      console.log(rows);
```
Comment (mallionaire) [inline model/reviews.js]: Please make sure you remove console.logs

**Review by mallionaire — CHANGES_REQUESTED** [review]
Lovely, you'll need to consider what might happen if you had a review that didn't have any comment. 
Have you written a test for that? 👀

File: model/reviews.js (hunk 1 above)
Comment (adnaani) [reply model/reviews.js]: I did considered that test after you told me but I didnt save it :( 
And I will get rid of the logs!

**Review by popatre — APPROVED** [review]
Looks good

**Pull request merged**